김진/231205/문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,7 @@
+function solution(s) {
+    let answer = [];
+    for (let i=0;i<s.length;i++) {
+      answer.push(s[i]);
+    }
+    return answer.sort().reverse().join('');
+  }
```

Comment:
```suggestion
function solution(s) {
    return s.split('').sort().reverse.().join(''); 
  }
```
굳이 answer에 안넣고 이렇게 바로 해도 되긴 합니다.